std_msgs/Header header
time image_time
PlateResult[] plates
sensor_msgs/RegionOfInterest[] regions_of_interest
